### PR TITLE
Updated oauth sso documentation

### PR DESCRIPTION
**Pull request by jsanchez-slingr**

### CODE REVIEW

File: content/en/dev-reference/security/single-sign-on.md [hunk 1]
```diff
@@ -487,3 +487,116 @@ If you decide to provide a custom mapping, here is the context of the script:
...
+      }
+    }
+  }
+  const firstName = userDataResponse.data.customer.firstName || 'Default first name';
+  const lastName = userDataResponse.data.customer.lastName || 'Default last name';
+  if (!email) throw('Missing required fields: email') // you might throw an exception for firstName and lastName
+  userRecord.field('firstName').val(firstName);
+  userRecord.field('lastName').val(lastName)
+  userRecord.field('email').val(email)
+  return sys.data.save(userRecord);
```
Comment (pasaperez-slingr) [inline content/en/dev-reference/security/single-sign-on.md:598]: I would explicitly indicate that the userRecord must be returned.
```suggestion
  sys.data.save(userRecord);
  return userRecord;
```

File: content/en/dev-reference/security/single-sign-on.md [hunk 2]
```diff
@@ -487,3 +487,116 @@ If you decide to provide a custom mapping, here is the context of the script:
...
+        return userRegistered;
+      }
+    }
+  }
+  const firstName = userDataResponse.data.customer.firstName || 'Default first name';
+  const lastName = userDataResponse.data.customer.lastName || 'Default last name';
+  if (!email) throw('Missing required fields: email') // you might throw an exception for firstName and lastName
+  userRecord.field('firstName').val(firstName);
+  userRecord.field('lastName').val(lastName)
+  userRecord.field('email').val(email)
```
Comment (pasaperez-slingr) [inline content/en/dev-reference/security/single-sign-on.md:597]: I would add an optional field to indicate that it is a good time to update the user.
```suggestion
  userRecord.field('firstName').val(firstName);                              // Required
  userRecord.field('lastName').val(lastName);                              // Required
  userRecord.field('email').val(email);                                           // Required
  userRecord.field('currentlyWorkingOn').val("Google");              // Optional
```

File: content/en/dev-reference/security/single-sign-on.md [hunk 3]
```diff
@@ -487,3 +487,116 @@ If you decide to provide a custom mapping, here is the context of the script:
   <br>
 
   ---
+
+## **OAuth**
+
+OAuth is an open-standard authorization protocol that allows secure third-party access to a user's resources without sharing their credentials (see [OAuth 2.0](https://en.wikipedia.org/wiki/OAuth)).
```
Comment (pasaperez-slingr) [inline content/en/dev-reference/security/single-sign-on.md:493]: Let's use the official site of Oauth
```suggestion
OAuth is an open-standard authorization protocol that allows secure third-party access to a user's resources without sharing their credentials (see [OAuth 2.0](https://auth0.com/intro-to-iam/what-is-oauth-2)).
```

**Review by pasaperez-slingr — APPROVED** [review]
Looks good, plese commit the suggestions. @jsanchez-slingr

**Comment by sebasmoyano** [comment]
Looks good. #approve

**Pull request merged**